Tests/MessagingInApp/Views/InAppMessageViewTest.swift

```diff
@@ -93,9 +90,23 @@ class InAppMessageViewTest: UnitTest {
         queueMock.getInlineMessagesReturnValue = [givenInlineMessage]
 
         let inlineView = InAppMessageView(elementId: givenInlineMessage.elementId!)
-        await onDoneRenderingInAppMessage(givenInlineMessage)
+        await onDoneRenderingInAppMessage(givenInlineMessage, insideOfInlineView: inlineView)
+
+        XCTAssertTrue(isInlineViewVisible(inlineView))
+        XCTAssertEqual(getInAppMessage(forView: inlineView), givenInlineMessage)
+    }
+
+    @MainActor
+    func test_displayInAppMessage_givenMultipleMessagesInQueue_expectDisplayFirstMessage() async {
```

Comment:
Added missing test to check what message is displayed when the local queue has multiple messages in it.